File: NetworkProviders/Hot-API-Client/Hot.API.Client/Models/BulkEvdResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hot.API.Client.Models
{
    public class BulkEvdResponse
    {
        public int ReplyCode{get; set;} = 0;
        public string ReplyMsg{get; set;} = "";
        public string ReplyMessage { 
            get { return ReplyMsg; } 
            set { ReplyMsg = value; }
        }
        public decimal WalletBalance{get; set;} = 0 ;
        public decimal Amount{get; set;} = 0;
        public decimal Discount{get; set;} = 0;
        public List<string> Pins{get; set;} 
        public string AgentReference{get; set;} = "";
        public int RechargeID{get; set;} = 0;
    }
}

File: Customer APIs/hot.backend.all/hot.api.backend/Hot.Api.Backend/Models/BulkEvdResponse.vb
Namespace Models
    Public Class BulkEvdResponse
        Public Sub New(agentReference As String)
            Me.AgentReference = agentReference
        End Sub

        Public Property ReplyCode As Integer = 0
        Public Property ReplyMsg As String = ""
        Public Property WalletBalance As Decimal = 0
        Public Property Amount As Decimal = 0
        Public Property Discount As Decimal = 0

        'Public Property Pins As New List(Of Hot.Lib.Models.PinRechargeModel)
        Public Property Pins As New List(Of String)

        Public Property AgentReference As String = ""
        Public Property RechargeID As Integer
    End Class
End Namespace
